spec/models/manageiq/providers/kubernetes/container_manager/metrics_capture/capture_context_spec.rb

```diff
@@ -26,7 +26,8 @@
                                                           :userid   => "_"}}]
     )
 
-    VCR.use_cassette("#{described_class.name.underscore}_refresh") do # , :record => :new_episodes) do
+    VCR.use_cassette("#{described_class.name.underscore}_refresh",
+                     :match_requests_on => [:path,]) do # , :record => :new_episodes) do
```

Comment:
I think its better to stay there, this way we won't be depended on the host name.